crates/core/src/commands.rs

```diff
@@ -12,6 +12,7 @@ pub mod dump;
 pub mod forget;
 pub mod init;
 pub mod key;
+pub mod lock;
```

Comment:
We don't expose anything via our API? There are no changes to `lib.rs`.

Reply:
We only expose new things on `RepositoryOptions` and a new method on `Repository`..